src/Components/WebAssembly/WebAssembly/test/Hosting/Logging/WebAssemblyLoggerFactoryTest.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I'll be working on adding more tests here but this is ready for review in the meanwhile.